src/main/java/com/denizenscript/denizencore/tags/core/UtilTagBase.java

```diff
@@ -952,6 +953,20 @@ else if (attribute.startsWith("format", 2) && attribute.hasParam()) {
         tagProcessor.registerTag(ElementTag.class, "stack_trace", (attribute, object) -> {
             return new ElementTag(DebugInternals.getFullExceptionMessage(new RuntimeException("TRACE"), false));
         });
+
+        // <--[tag]
+        // @attribute <util.mongo_connections>
+        // @returns ElementTag
+        // @description
+        // Returns a list of all Mongo connections opened by <@link command mongo>
```

Comment:
period at end.
Also add this tag next to the matching sql/redis tags so they're together.